lib/config/error-codes/api.json

```diff
@@ -66,25 +81,25 @@
       "errors": {
         "token_force_expire": {
           "code": 1,
-          "message": "Error while forcing token expiration",
+          "message": "Error while forcing token expiration.",
```

Comment:
Why not, but in that case you have to be consistent and apply that modification for all messages.